subprojects/core/src/integTest/groovy/org/gradle/api/internal/tasks/SnapshotTaskInputsOperationIntegrationTest.groovy

```diff
@@ -44,7 +47,7 @@ class SnapshotTaskInputsOperationIntegrationTest extends AbstractIntegrationSpec
 
         then:
         result.buildCacheKey != null
-        result.inputHashes.keySet() == ['input1', 'input2'] as Set
+        result.inputHashes.keySet() == ['bean.class', 'input1', 'input2'] as Set
```

Comment:
Hm, I didn't realize we still add a `.class` property. Is this still required? Also, why do we need to mention it here now?

Reply:
It is not required. I think we decided to leave it in for backwards compatibility.
It is present now, since the task has a `bean` nested property now, so it is always tracked as an input - even if the bean is not set.